docs/source/nn.rst

```diff
@@ -32,35 +32,35 @@ Augmentation Layers
     :nosignatures:
     :template: classtemplate.rst
 
-    kornia.nn.RandomHorizontalFlip
-    kornia.nn.RandomVerticalFlip
-    kornia.nn.CenterCrop
-    kornia.nn.ColorJitter
-    kornia.nn.RandomAffine
-    kornia.nn.RandomBoxBlur
-    kornia.nn.RandomChannelShuffle
-    kornia.nn.RandomCrop
-    kornia.nn.RandomElasticTransform
-    kornia.nn.RandomEqualize
-    kornia.nn.RandomErasing
-    kornia.nn.RandomFisheye
-    kornia.nn.RandomGaussianBlur
-    kornia.nn.RandomGaussianNoise
-    kornia.nn.RandomGrayscale
-    kornia.nn.RandomHorizontalFlip
-    kornia.nn.RandomInvert
-    kornia.nn.RandomMotionBlur
-    kornia.nn.RandomPerspective
-    kornia.nn.RandomPosterize
-    kornia.nn.RandomResizedCrop
-    kornia.nn.RandomRotation
-    kornia.nn.RandomSharpness
-    kornia.nn.RandomSolarize
-    kornia.nn.RandomThinPlateSpline
-    kornia.nn.RandomVerticalFlip
-    kornia.nn.PadTo
-    kornia.nn.Normalize
-    kornia.nn.Denormalize
+    nn.RandomHorizontalFlip
```

Comment:
Are we going to have duplicate content for augmentations?

Reply:
Only titles. This will generate a list than full docs.

Reply:
Should we remove then any module-like from the existing docs ?

Reply:
Not sure on that. I would take the nn modules as a shortcut.